src/App.tsx

```diff
@@ -2,37 +2,23 @@ import React, { lazy } from 'react'
 
 import './App.css'
 import { HashRouter, Route, Switch } from 'react-router-dom'
-import ElectronStore from 'electron-store'
 import Sidebar from 'src/components/UI/Sidebar'
 import Login from './screens/Login'
 import WebView from './screens/WebView'
 
-const Store = window.require('electron-store')
-
 const Library = lazy(() => import('./screens/Library'))
 const Settings = lazy(() => import('./screens/Settings'))
 const GamePage = lazy(() => import('./screens/Game/GamePage'))
 const WineManager = lazy(() => import('./screens/WineManager'))
 
 function App() {
-  const configStore: ElectronStore = new Store({
-    cwd: 'store'
-  })
-  const gogStore: ElectronStore = new Store({
-    cwd: 'gog_store'
-  })
-
-  const user = configStore.has('userInfo') || gogStore.has('credentials')
-
   return (
     <div className="App">
       <HashRouter>
         <Sidebar />
         <main className="content">
           <Switch>
-            <Route exact path="/">
-              {user && <Library />}
-            </Route>
+            <Route exact path="/" component={Library} />
```

Comment:
we don't need to check for the user here, the SidebarLinks component is redirecting to the login if no user logged in (though that shouldn't happen in the SidebarLinks either, it makes more sense to do that check in the GlobalState component)